include/deal.II/grid/tria_accessor.h

```diff
@@ -1290,7 +1290,20 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
                           const bool use_laplace_transformation=false) const;
 
   /**
-   * Barycenter of the object.
+   * Return the barycenter of the object. The barycenter for an object $K$
+   * of dimension $d$ in $D$ space dimensions is given by the $D$-dimensional
+   * vector $\mathbf x_K$ defined by
+   * @f{
+   *   \mathbf x_K = \frac{1}{|K|} \int \mathbf x \; \textrm{d}^dx
```

Comment:
also: while ``\textrm{d}^dx`` is correct, I find the notation confusing, maybe just write ``dx`` ?